### ISSUE TITLE
Subsonic API - getUser

### ISSUE THREAD

**Issue by dhsc19** [ampache/ampache#770]
DSub mobile client uses getUser to check user permissions in order to allow sharing.

I'm seeing this response from Ampache server:

![screenshot from 2015-03-15 00 35 13](https://cloud.githubusercontent.com/assets/10291198/6655107/8094b902-caab-11e4-8970-647de42d7746.png)

**Comment by daneren2005** [comment]
That was just looked up incorrectly.  The response I got is:

\<subsonic-response type="ampache" version="1.11.0" status="ok"\>\<user username="subsonic" email="xxxx" scrobblingEnabled="false" adminRole="false" settingsRole="true" downloadRole="true" playlistRole="true" coverArtRole="false" commentRole="false" podcastRole="false" streamRole="true" jukeboxRole="false" shareRole="false"/\>\</subsonic-response\>

Does Ampache have somewhere where you set user permissions?  If they are just hard coded, then shareRole needs to be changed to true.

**Comment by dhsc19** [comment]
I've confirmed that Admin -> Server Config -> Interface -> Allow Share is set to "Enable" for access level "User"

**Comment by Afterster** [comment]
Ok (thank you for xml output daneren2005), but are you sure? shareRole should be set to true if your user has share rights. Moreover this is working on my side.
Please double check your user rights, apply share access to everybody or try with another user.

**Comment by dhsc19** [comment]
The only place I found "Allow Share" setting is in the Server Config -> System.  And that is currently set to Enable for Access Level "User".  I can confirm changing this setting changes the share access on the web interface correctly.

I tried switching access level to Guest and also checking the Apply to All box.  I'm still seeing shareRole="false" in the xml output.  I also tried looking in the user preference settings and I did not find any other setting related to sharing.  Am I looking in the right place?
